app/src/main/java/org/simple/clinic/illustration/IllustrationSync.kt

```diff
@@ -0,0 +1,22 @@
+package org.simple.clinic.illustration
+
+import io.reactivex.Completable
+import io.reactivex.Single
+import org.simple.clinic.sync.ModelSync
+import org.simple.clinic.sync.SyncConfig
+import javax.inject.Inject
+import javax.inject.Named
+
+class IllustrationSync @Inject constructor(
+    private val illustrations: List<HomescreenIllustration>,
```

Comment:
Can we inject the repository here instead and read the illustrations from the method? The reasons are the same as described above.

Reply:
Done